07-routing-gossip.md

```diff
@@ -445,6 +445,7 @@ The origin node:
   `announce_channel` bit was not set).
     - MUST NOT forward such a `channel_update` to other peers, for privacy
     reasons.
+    - MUST NOT send a `channel_update` before `funding_locked` has been received.
```

Comment:
Should this be clarified to only apply to such channel updates that are just destined for a channel peer?

Reply:
Thank you.

a `channel_update` ==> a created `channel_update`